cmd/load-runner/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"regexp"
+	"time"
+)
+
+type config struct {
+	duration       string
+	parsedDuration time.Duration
+	namespace      string
+	loadName       string
+	chartFilesPath string
+}
+
+const (
+	defaultReleaseNamePrefix = "rudder-load"
+	defaultChartFilesPath    = "./artifacts/helm"
+)
+
+func main() {
+	cfg := parseFlags()
+	if err := validateInputs(cfg); err != nil {
```

Comment:
If you move this stuff into `run` then you avoid the repetition of having to check for `err` and do `os.Exit(1)`.

Reply:
done